packages/clay-charts/src/Geomap.js

```diff
@@ -263,20 +262,10 @@ Geomap.STATE = {
 		selected: '#4b9bff',
 		value: 'pop_est',
 	}),
-
-	/**
-	 * Geo-json data
-	 * @instance
-	 * @memberof Geomap
-	 * @type {?Function|?Object|?String}
-	 * @default undefined
-	 */
-	data: Config.oneOfType([
-		Config.func(),
-		Config.object(),
-		Config.string(),
-	]).required(),
 };
 
+Object.assign(Geomap.prototype, DataComponent);
+Object.assign(Geomap.STATE, DataComponent.STATE);
```

Comment:
Same as before?

Reply:
Here, I'm mutating `Geomap.prototype` which is what I want and doing the same which `Geomap.STATE` which is another object so I don't really want to mix the two, but I'm not modifying `DataComponent` or `DataComponent.STATE`, did I miss anything?

Reply:
It probably makes more sense to have `Geomap` extend from `DataComponent` since itself extends `Component`, and just copy the `STATE` over. What do you think?